docs/DeveloperGuide.adoc

```diff
@@ -349,6 +395,10 @@ _{More to be added}_
 .  Should work on any <<mainstream-os,mainstream OS>> as long as it has Java `11` or above installed.
 .  Should be able to hold up to 1000 persons without a noticeable sluggishness in performance for typical usage.
 .  A user with above average typing speed for regular English text (i.e. not code, not system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.
+.  Should respond within 2 seconds
+.  Should be easy to use for users who are not IT-savvy
```

Comment:
This NFR's testability seems a little ambiguous. Maybe you could add a few details